tsconfig.tsbuildinfo

```diff
@@ -1 +0,0 @@
-{"version":"4.4.4"}
```

Comment:
why delete this?

Reply:
I revert this, and other unintentional changes.